src/components/ErrorMessage/ErrorMessage.tsx

```diff
@@ -0,0 +1,48 @@
+import React, { FC, useEffect } from 'react';
+import classNames from 'classnames';
+import { ErrorType } from '../../types/Error';
+
+type Props = {
+  error: ErrorType;
+  setError: React.Dispatch<React.SetStateAction<ErrorType>>
+};
+
+export const ErrorMessage: FC<Props> = React.memo(({ error, setError }) => {
+  const handleErrorMessageHidden = () => {
+    setError(ErrorType.NONE);
+  };
+
+  useEffect(
+    () => {
+      if (error === ErrorType.NONE) {
+        return () => {};
+      }
```

Comment:
looks strange actually) why just do not `return;` ?

Reply:
Well, it because of the linter). Though i found another way to fix it(return undefined).